README.md

```diff
@@ -329,6 +329,28 @@ class Product {
 }
 ```
 
+If you have a list of acceptable status codes you can also pass an array to the
+```statusCode``` option:
+
+```php
+<?php
+namespace CircleBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Circle\DoctrineRestDriver\Annotations as DataSource;
+
+/**
+ * This annotation marks the class as managed entity:
+ *
+ * @ORM\Entity
+ * @ORM\Table("products")
+ * @DataSource\Select("http://www.yourSite.com/api/products/findOne/{id}", statusCode={200, 203, 404})
```

Comment:
Oh you will hate me for this one :-D
statusCode is singular, but now we can add multiple codes (plural).

The API should be changed to statusCodes .... :( I'm sorry!

Reply:
I finally got around to adding to this PR! Your previous request should now be fulfilled.